js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -102,8 +102,8 @@ export const AssetNode: React.FC<{
             </>
           )}
           <StatsRow>
-            <span>Latest Run</span>
-            <CaptionMono>
+            <span>Latest&nbsp;Run</span>
```

Comment:
Yeah, this was a weird one - if you opened the SVG without our fonts loaded, it would render wrong because we apparently have the box sized precisely. To make sure it never wraps when you open the SVG I made it a nonbreaking space here 😬